test/spec/pinball_wizard_spec.coffee

```diff
@@ -267,6 +267,23 @@ define ['pinball_wizard'], (pinball) ->
       urlParam = '?foo=bar&pinball=a,b&bar'
       expect(pinball._urlValues(urlParam)).toEqual(['a','b'])
 
+  describe '#activatePermanently', ->
```

Comment:
Can we combine this describe block with the next? I have a typo on the method name :-/